api/model/src/main/java/org/projectnessie/model/ContentResponse.java

```diff
@@ -44,6 +49,11 @@ static ImmutableContentResponse.Builder builder() {
   @Value.Parameter(order = 2)
   Reference getEffectiveReference();
 
+  /** Additional content related information, if any. */
+  @JsonInclude(Include.NON_EMPTY)
+  @JsonView(Views.V2.class)
+  List<ContentMetadata> getMetadata();
```

Comment:
Could it be useful to have a list of metadata types in the corresponding _request_ object?

Is it meaningful for a client to fetch (some) metadata _without_ a `Content` object?

Reply:
Not sure whether the `getMetadata()` in `ContentResponse` will be of any use short-term, but might be mid-term. Just would like to be prepared for the unlikely case.

Reply:
Well, should we support fetching content without the metadata (overhead) then?